### ISSUE TITLE
APP Swipe between pictures

### ISSUE THREAD

**Issue by alanjcook** [akshayvernekar/Brilliant_pad_issue_management#20]
• user should be able to swipe between pictures in log
• home screen picture should have date/time/image size/resolution/compression details

**Comment by cumulations** [comment]
@alanjcook  we need to think about swipe left and right usecases.
 When the event next to the current lookin event is an advance pad event, then the full-screen mode doesn't make sense right?

Or are you suggesting to support only the "animal entry/exit event and lookin event" for swipe left and right in fullscreen mode?

**Comment by alanjcook** [comment]
Correct..swipe to display next/previous picture in full screen mode.
